packages/ui-library/src/components/select/index.ts

```diff
@@ -108,7 +115,7 @@ export class BlrSelect extends LitElementCustom {
         this.sizeVariant,
       ]).toLowerCase() as SizesType;
 
-      const modifiedIcon = this.icon ? this.icon : 'blrChevronDown';
+      const modifiedIcon = this.icon;
```

Comment:
This is used in only one place and also you can directly use the this.icon so this variable can be removed.

Reply:
done